api/routes/project.js

```diff
@@ -1,8 +1,41 @@
+var express = require('express');
+var router = express.Router();
 var Project = require('../models/Projects');
 
-newProject = function() {
+// retrieves all projects
```

Comment:
We need to add `Pagination` to this request and we also need to `SORT` the results and we need to decide by what means we should sort out.